modules/gathering-log-data.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * updating/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-log-data_{context}"]
+= Gathering log data
```

Comment:
This might be better framed as gathering data for support, if that's the purpose of the log data

```suggestion
= Gathering log data for a support case
```

Reply:
Agreed - ty :)